cmd/kdk/kdk.go

```diff
@@ -15,24 +15,17 @@
 package cmd
 
 import (
-	"context"
-	"io/ioutil"
 	"os"
-	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/cisco-sso/kdk/internal/app/kdk"
-	"github.com/docker/docker/client"
+	"github.com/cisco-sso/kdk/pkg/kdk"
 	"github.com/ghodss/yaml"
-	"github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"io/ioutil"
```

Comment:
Should: Move `"io/util"` up underneath `"os"` to group together the core library imports.

Reply:
done